recipes/tssmsr/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, MSBuild, tools
+from conans.errors import ConanInvalidConfiguration
+from glob import glob
+
+
+class TssMsrConan(ConanFile):
+    name = "tssmsr"
+    description = "TPM Software Stack (TSS) implementations from Microsoft"
+    topics = ("tpm", "tss", "crypto", "microsoft")
+    homepage = "https://github.com/microsoft/TSS.MSR"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    settings = "os", "compiler", "arch", "build_type"
+    generators = "make"
+    exports_sources = "patches/*"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return os.path.join(self._source_subfolder, "TSS.CPP")
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "14",
+            "gcc": "5",
+            "clang": "3.9",
+            "apple-clang": "8",
+        }
+
+    def configure(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, self._minimum_cpp_standard)
+        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
+        if not min_version:
+            self.output.warn("{} recipe lacks information about the {} compiler support.".format(
+                self.name, self.settings.compiler))
+        else:
+            if tools.Version(self.settings.compiler.version) < min_version:
+                raise ConanInvalidConfiguration("{} requires C++{} support. The current compiler {} {} does not support it.".format(
+                    self.name, self._minimum_cpp_standard, self.settings.compiler, self.settings.compiler.version))
+
+    def requirements(self):
+        # VC builds link against openssl included in the sources
```

Comment:
What's different about the inline one? Typically we use a small patch to make sure it uses the Conan one

Reply:
It just looks like an old one, you should be able to remove it and include the msbuild props generated from conan 🤞